app/services/data/get-advance-claims-over-specified-date-and-claim-expense-amount.js

```diff
@@ -0,0 +1,16 @@
+const config = require('../../../knexfile').asyncworker
```

Comment:
Might be worth renaming this module, the name seems a little confusing to me, eg. get-advance-claims-total-expense-approved-cost-before-date